dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -830,3 +837,50 @@ CWWKD1082.entity.classes.missing.explanation=Repositories use entity classes \
  that must be defined in the persistence unit.
 CWWKD1082.entity.classes.missing.useraction=Include all entity classes \
  in the persistence unit.
+
+CWWKD1083.dup.method.param=CWWKD1083E: Multiple parameters of the {0} method \
+ of the {1} repository interface correspond to the named parameter: {2}. \
+ You can annotate one method parameter with {3} to specify the query language \
+ named parameter. Alternatively, you can enable the -parameters compile option \
+ and define one method parameter of the form, {4}, which does not require the \
+ Param annotation.
```

Comment:
Multiple parameters in the {0} method of the {1} repository interface correspond to the named parameter: {2}. You can annotate one method parameter with {3} to specify the query language named parameter. Alternatively, you can enable the -parameters compile option and define one method parameter in the form, {4}, which does not require the Param annotation.

Reply:
updating as suggested